lib/Config.js

```diff
@@ -1220,145 +1175,6 @@ class Config extends EventEmitter {
             maxStaleness,
             enableInflights,
         };
-        if (config.utapi) {
```

Comment:
we should keep the code, only need to replace the condition to make sure it never happens:


```suggestion
        if (false && config.utapi) {
```

(and we can keep the `parseUtapiReindex` function as well)